Stanford/JumpstartEngineering/Install/Layouts/PicalHomepage.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file
+ * Configure PICAL homepage layouts.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class PicalHomepage extends \AbstractInstallTask {
+
+  /**
+   * Configure PICAL homepage layouts.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+
+    $default = 'stanford_jumpstart_home_morris';
+    variable_set('stanford_jumpstart_home_active_body_class', 'stanford-jumpstart-home-morris');
+    $context_status = variable_get('context_status', array());
+    $homecontexts = stanford_jumpstart_home_context_default_contexts();
+    $names = array_keys($homecontexts);
+    // Enable these JSE layouts for use by site owners.
+    $enabled['stanford_jumpstart_home_hoover'] = 1;
+    $enabled['stanford_jumpstart_home_morris'] = 1;
+    $enabled['stanford_jumpstart_home_terman'] = 1;
+    $enabled['stanford_jumpstart_home_pettit'] = 1;
+    // Disable these layouts.
+    unset($enabled['stanford_jumpstart_home_lomita']);
```

Comment:
I don't think these are even available so I don't think we need to disable these.